javascript/src/main/pages/Courses/Courses.js

```diff
@@ -17,6 +19,12 @@ const Courses = () => {
     ["/api/public/courses", getToken],
     fetchWithToken
   );
+  const deleteCourse = buildDeleteCourse(getToken, mutateCourses);
+  const { data: active } = useSWR(
+    "/api/public/filter",
+    fetchWithoutToken
+  );
+  console.log(active);
```

Comment:
I wonder whether we still need this? Unless we're logging something that should be seen even in production, we should probably remove this.